deploy/README.md

```diff
@@ -161,16 +162,57 @@ kubectl -n sumologic create secret generic sumologic \
   --from-literal=endpoint-events=$ENDPOINT_EVENTS
 ```
 
-Apply `fluentd-sumologic.yaml` manifest with following command:
+Get the `fluentd-sumologic.yaml` manifest with following command:
 
 ```sh
 curl https://raw.githubusercontent.com/SumoLogic/sumologic-kubernetes-collection/master/deploy/kubernetes/fluentd-sumologic.yaml.tmpl | \
-sed 's/\$NAMESPACE'"/sumologic/g" | \
+sed 's/\$NAMESPACE'"/sumologic/g"
+```
+
+You can customize the provided YAML file if needed, including configuring fields for [events](#events). Once done run the following command to apply the `fluentd-sumologic.yaml` manifest.
+
+```sh
 kubectl -n sumologic apply -f -
```

Comment:
when this command is executed separately, we can't use the `-` at the end since there is no piped output to it. instead the file name needs to be explicitly specified
```
kubectl -n sumologic apply -f fluentd-sumologic.yaml
```